code/__DEFINES/rust_g.dm

```diff
@@ -0,0 +1,40 @@
+/// Locator for the RUSTG DLL or SO depending on system type
+#define RUST_G (world.system_type == UNIX ? "./librust_g.so" : "./rust_g.dll")
+
+// Defines for internal job subsystem //
+#define RUSTG_JOB_NO_RESULTS_YET "NO RESULTS YET"
+#define RUSTG_JOB_NO_SUCH_JOB "NO SUCH JOB"
+#define RUSTG_JOB_ERROR "JOB PANICKED"
+
+// DMI related operations //
+#define rustg_dmi_strip_metadata(fname) call(RUST_G, "dmi_strip_metadata")(fname)
+#define rustg_dmi_create_png(path, width, height, data) call(RUST_G, "dmi_create_png")(path, width, height, data)
+
+#define rustg_noise_get_at_coordinates(seed, x, y) call(RUST_G, "noise_get_at_coordinates")(seed, x, y)
+
+// Git related operations // - AA TODO
```

Comment:
Why the TODOs here? They are not yet used but are they usable? Is RUST not yet able to provide them? Or is it just for yourself? If it's the latter than I'd just remove it to reduce confusion. Or explain what there is todo

Reply:
TODOs for what I am working on, and also to dissuade other people using them yet due to their finnicky nature (These are things in my TODO list which need to be very specifically done)

Reply:
Then include that info there please. A random TODO in a public repo is a bit vague